File: src/types/BusiestPeriod.java
package types;

import java.time.LocalTime;

public class BusiestPeriod {
    private LocalTime startTime;
    private LocalTime endTime;
    private int numberOfDrivers;

    public BusiestPeriod(LocalTime startTime, LocalTime endTime, int numberOfDrivers) {
        this.startTime = startTime;
        this.endTime = endTime;
        this.numberOfDrivers = numberOfDrivers;
    }

    public LocalTime getStartTime() {
        return startTime;
    }

    public LocalTime getEndTime() {
        return endTime;
    }

    public int getNumberOfDrivers() {
        return numberOfDrivers;
    }
}

File: bin/types/BusiestPeriod.cs
namespace Types
{
    public class BusiestPeriod
    {
        private readonly DateTime startTime;
        private readonly DateTime endTime;
        private readonly int numberOfDrivers;

        public BusiestPeriod(DateTime startTime, DateTime endTime, int numberOfDrivers)
        {
            this.startTime = startTime;
            this.endTime = endTime;
            this.numberOfDrivers = numberOfDrivers;
        }

        public DateTime StartTime => startTime;

        public DateTime EndTime => endTime;

        public int NumberOfDrivers => numberOfDrivers;

        public TimeSpan Duration => endTime - startTime;
    }
}
